src/amo/components/UserProfileEdit/index.js

```diff
@@ -53,10 +55,11 @@ type Props = {|
   i18n: I18nType,
   isUpdating: boolean,
   lang: string,
-  // The routing `params` prop is used in `mapStateToProps()`.
-  // eslint-disable-next-line react/no-unused-prop-types
-  params: {| username: string |},
-  router: ReactRouterType,
+  match: {|
+    ...ReactRouterMatchType,
+    params: {| username: string |},
+  |},
+  history: ReactRouterHistoryType,
```

Comment:
Thanks, I'll try to apply these changes everywhere.

I filed https://github.com/mozilla/eslint-plugin-amo/issues/8 because I am a bit tired of these comments in general. Prettier does not fix this unfortunately, but hopefully we'll have a ESLint rule soon 🙏